### ISSUE TITLE
Help! When I use toltec to install rm2fb, I meet an error.

### ISSUE THREAD

**Issue by ponymushama** [matteodelabre/toltec#580]
Job for rm2fb.service failed because the control process exited with error code.
See "systemctl status rm2fb.service" and "journalctl -xe" for details.
Collected errors:
 * pkg_run_script: package "display" postinst script returned status 1.
 * opkg_configure: display.postinst returned 1.

**Comment by Eeems** [comment]
What OS version are you running? Also, did you get the output of the commands it provided to get more error information?

**Comment by LeifAndersen** [comment]
For whatever its worth, I also have this problem.

I'm currently on 2.12.2.573, but this has also been an issue I've had going back to ~2.9 (which I never resolved).

**Comment by Eeems** [comment]
> For whatever its worth, I also have this problem.
> 
> I'm currently on 2.12.2.573, but this has also been an issue I've had going back to ~2.9 (which I never resolved).

[As per the website, we don't currently support 2.12.2.573](https://toltec-dev.org/#install-toltec). Likely you've always tried installing toltec before we support the OS version you are on.

**Comment by rsaavedn** [comment]
> > For whatever its worth, I also have this problem.
> > I'm currently on 2.12.2.573, but this has also been an issue I've had going back to ~2.9 (which I never resolved).
> 
> [As per the website, we don't currently support 2.12.2.573](https://toltec-dev.org/#install-toltec). Likely you've always tried installing toltec before we support the OS version you are on.

What is the expected date for 2.12.2.573 support ?

**Comment by LeifAndersen** [comment]
> [As per the website, we don't currently support 2.12.2.573](https://toltec-dev.org/#install-toltec). Likely you've always tried installing toltec before we support the OS version you are on.

Ya, that makes sense.

> What is the expected date for 2.12.2.573 support ?

FWIW, it does seem to work if you switch to the testing branch. (I do wish there was a way to get just one package from testing, and the rest from stable. But oh well.)

**Comment by Eeems** [comment]
> > > For whatever its worth, I also have this problem.
> > > I'm currently on 2.12.2.573, but this has also been an issue I've had going back to ~2.9 (which I never resolved).
> > 
> > [As per the website, we don't currently support 2.12.2.573](https://toltec-dev.org/#install-toltec). Likely you've always tried installing toltec before we support the OS version you are on.
> 
> What is the expected date for 2.12.2.573 support ?

 A couple weeks. Still waiting on a couple branches to be merged into testing and then it will be in the next weekly merge window after the merge.

**Comment by ponymushama** [comment]
> What OS version are you running? Also, did you get the output of the commands it provided to get more error information?

OS is 2.12.2.573.
I didn't get the output of the commands.

**Comment by ponymushama** [comment]
> > For whatever its worth, I also have this problem.
> > I'm currently on 2.12.2.573, but this has also been an issue I've had going back to ~2.9 (which I never resolved).
> 
> [As per the website, we don't currently support 2.12.2.573](https://toltec-dev.org/#install-toltec). Likely you've always tried installing toltec before we support the OS version you are on.

I have installed toltec successfully, but I cannot install any package by now.

**Comment by Eeems** [comment]
We don't support the version of the OS that you are running. You will want to uninstall until we release support for it.
`toltecctl uninstall`

**Comment by ponymushama** [comment]
> We don't support the version of the OS that you are running. You will want to uninstall until we release support for it. `toltecctl uninstall`

Thx. I will uninstall it for now.
